scripts/node_builder_glibc_env.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -eu
+
+export DEBIAN_FRONTEND=noninteractive
+apt update -y && apt install -y curl g++ python3 make gcc git
```

Comment:
```suggestion
apt update -y && apt install -y --no-install-recommends curl g++ python3 make gcc git
```

best practice is to add these everywhere, to reduce downloaded & installed size